src/coreComponents/schema/schema.xsd

```diff
@@ -222,10 +222,18 @@
 				</xsd:unique>
 			</xsd:element>
 			<xsd:element name="Functions" type="FunctionsType" maxOccurs="1">
+				<xsd:unique name="FunctionsCompositeFunctionUniqueName">
```

Comment:
Some of these schema changes appear unrelated to the code changes. Are these perhaps part of a different PR?